portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -33,6 +33,16 @@ async function getData() {
   }
 }
 
+/**
+ * Forwards POST request to delete-data servlet and refreshes portfolio with
```

Comment:
"Forward POST request to delete-data servlet" explains what the code is doing and not what the method does. I would rephrase as:

Deletes comments and refreshes portfolio with updated comments.

Reply:
done